terraform/asg-aarch64.tf

```diff
@@ -0,0 +1,69 @@
+resource "aws_autoscaling_group" "aarch64prod-mixed" {
+  lifecycle {
+    create_before_destroy = true
+  }
+
+  default_cooldown          = local.cooldown
+  health_check_grace_period = local.grace_period
+  health_check_type         = "ELB"
+  max_size                  = 4
+  min_size                  = 1
+  name                      = "aarch64prod"
+  vpc_zone_identifier       = local.subnets
+
+  mixed_instances_policy {
+    instances_distribution {
+      // This base value is zero so we don't have any non-spot instances. We may wish to bump this if we have issues
+      // getting spot capacity.
+      on_demand_allocation_strategy            = "lowest-price"
+      on_demand_base_capacity                  = 0
+      on_demand_percentage_above_base_capacity = 0
+      spot_allocation_strategy                 = "price-capacity-optimized"
+      spot_instance_pools                      = 0
+    }
+    launch_template {
+      launch_template_specification {
+        launch_template_id = aws_launch_template.CompilerExplorer-aarch64prod.id
+        version            = "$Latest"
+      }
+      override {
+        instance_type = "r7g.medium"
```

Comment:
https://instances.vantage.sh/aws/ec2/r7g.medium

I built the packer image with a Graviton3 cpu, I don't really know if we can also add Grav2 to this mix
But if possible we could also add https://instances.vantage.sh/aws/ec2/c6g.xlarge
But it might just be better to stick to 1 type for now

Reply:
nice